components/script/dom/webidls/MutationRecord.webidl

```diff
@@ -0,0 +1,16 @@
+[Pref="dom.mutation_observer.enabled", Exposed=Window]
+interface MutationRecord
+ {
+  readonly attribute DOMString type;
+  [SameObject]
+  readonly attribute Node target;
+  [SameObject]
+  readonly attribute NodeList addedNodes;
+  [SameObject]
+  readonly attribute NodeList removedNodes;
+  readonly attribute Node? previousSibling;
+  readonly attribute Node? nextSibling;
+  readonly attribute DOMString? attributeName;
+  readonly attribute DOMString? attributeNamespace;
+  readonly attribute DOMString? oldValue;
```

Comment:
Rather than implementing stubs for all of these attributes, let's comment out all of them in this file except for `type` and `target`. We can uncomment them when we give them real implementations.